src/util/AppContextUtil/ShogunBootAppContextUtil.tsx

```diff
@@ -261,6 +263,8 @@ class ShogunBootAppContextUtil extends BaseAppContextUtil implements AppContextU
     tileLayer.set('type', layer.type);
     tileLayer.set('legendUrl', legendUrl);
     tileLayer.set('timeFormat', defaultFormat);
+    tileLayer.set('searchable', searchable);
+    tileLayer.set('searchConfig', searchConfig);
```

Comment:
cannot instances of [imageLayer](https://github.com/terrestris/react-geo-baseclient/blob/2e09445a3f3feb87c7b81d9cf406318283ab9208/src/util/AppContextUtil/ShogunBootAppContextUtil.tsx#L312) and [wmtsLayer](https://github.com/terrestris/react-geo-baseclient/blob/2e09445a3f3feb87c7b81d9cf406318283ab9208/src/util/AppContextUtil/ShogunBootAppContextUtil.tsx#L370) be searchable and have searchConfig as well?

Reply:
Right, added in https://github.com/terrestris/react-geo-baseclient/pull/751